calicoctl/calico_ctl/container.py

```diff
@@ -506,7 +506,8 @@ def get_ip_and_pool(ip):
             ip = result[1][0]
         pool = get_pool_or_exit(ip)
     elif ip is not None and '/' in ip:
-        pool = IPPool(ip)
+        cidr = IPNetwork(ip)
+        pool = get_pool_or_exit(IPAddress(cidr.first))
```

Comment:
And does this mean that `__eq__` method of `IPPool` object can only compare `cidr` of each other and just ignore `ipam`, `masquerade` ... If so, that would be a better solution